src/packages/@library/forms/text-editor/TextEditor.svelte

```diff
@@ -0,0 +1,282 @@
+<script>
+  import EditorJS from "@editorjs/editorjs";
+  import Header from "@editorjs/header";
+  import List from "@editorjs/list";
+  import InlineCode from "@editorjs/inline-code";
+  import CodeTool from "@editorjs/code";
+  import Underline from "@editorjs/underline";
+  import Paragraph from "@editorjs/paragraph";
+  import edjsParser from "editorjs-parser";
+  import { onMount } from "svelte";
+  export let value = "[]";
+  export let onInput;
+  export let id;
+  export let isReadOnly = false;
+  export let placeholder;
```

Comment:
initailized it with some default values to prevent undefined error if used in UI. type and comment missing!